dbusmock/templates/modemmanager.py

```diff
@@ -196,11 +207,13 @@ def AddSimpleModem(self):
     modem.AddProperties(MODEM_3GPP_IFACE, modem_3gpp_props)
 
     modem_cell_broadcast_props = {
+        "Channels": dbus.Array([], signature="(uu)"),
         "CellBroadcasts": dbus.Array([], signature="o"),
     }
     modem_cell_broadcast_methods = [
         ("List", "", "ao", listCbm),
         ("Delete", "o", "", deleteCbm),
+        ("SetChannels", "a(uu)", "", setChannels),
```

Comment:
OK, these match https://gitlab.freedesktop.org/mobile-broadband/ModemManager/-/commit/2878833f5c83fe112f579865a6d318ce8413f452

Let's see if the next MM upstream release will break the tests due to changing the output format. I actually wish it did, then we'd have a good reminder to check this new option :wink:

Reply:
I think it won't. We need to additionally run `mmcli -m any --cell-broadcast-status` for that so it won't break out of a sudden.